atc/exec/build/repository.go

```diff
@@ -20,13 +20,16 @@ type ArtifactName string
 //
 type Repository struct {
 	repo  map[ArtifactName]runtime.Artifact
-	repoL sync.RWMutex
+	repoL *sync.RWMutex
```

Comment:
I can undo this actually. I originally did it because my first implementation of `NewLocalScope` involved cloning the parent like this:

```go
clone := *repo
```

My editor threw a warning for this because it copied the mutex, so I changed the mutex to a pointer and reassigned it.

But then I ended up changing it to just `clone := NewRepository()` and assigning `.parent` after, so this change isn't needed anymore. I'll just undo it to prevent code churn.

Reply:
Cool, I don't really have a preference, was mostly just wondering if you knew something I didn't 😁